web-frontend/modules/core/components/auth/TOTPLogin.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div>
+    <svg
```

Comment:
I realize this deviates a little bit from the design, but we have not implemented the icon-only style for the various login pages. Woud you mind replacing this with the `<Logo>` component?

This page seems to have it's own css classes to style it properly. What do you think of making it consistent with how the `modules/core/pages/forgotPassword.vue`, `modules/core/pages/resetPassword.vue`, etc pages using the `auth__wrapper`, `auth__logo`, classes to render this page? I realize it won't be matching the design then properly, but when we then change the styling, it will be consistent on all pages.

Reply:
I changed it to reuse the existing layout. Because it is not an exact match and there is a lot of discrepancy between the designs it might not be that good. The TOTP login screen now looks a bit misaligned because of the full-size button and code inputs. I suggest we address it separately if desired. In that case it would be great to provide more details on how it should be handled.